src/Scene.h

```diff
@@ -0,0 +1,32 @@
+#ifndef SCENE_H
+#define SCENE_H
+
+#include "CGL/CGL.h"
+#include "../SchwarzschildBlackHoleEquation.h"
+#include <opencv2/opencv.hpp>
+#include <iostream>
+
+using namespace CGL;
+using namespace std;
+using namespace cv;
+
+struct Scene {
+  public:
+    Scene(Vector3D &CameraPosition, Vector3 &CameraLookAt, Vector3 &UpVector, float Fov, List<IHitable> hitables, float CurvatureCoeff, float AngularMomentum); // TODO: what should we use for List data structure
```

Comment:
Forgot to change some of these to Vector3Ds. 
Consider using vector `<IHitable *> &hitables` instead of the list here.
I think its good to make some of the structs here pass by reference instead of value.